product_state/models/product_state.py

```diff
@@ -24,7 +25,7 @@ class ProductState(models.Model):
         string="Number of products",
         compute="_compute_products_count",
     )
-
+    default = fields.Boolean("Default state")
```

Comment:
we should have an `@api.constrains` check to make sure we have only 1 default at time

Reply:
default = fields.Boolean("Default state", default=False)

Reply:
the [boolean](https://github.com/odoo/odoo/blob/14.0/odoo/fields.py#L1204) was inherited default = None from [Field](https://github.com/odoo/odoo/blob/14.0/odoo/fields.py#L234) and it is convert to False when we install the module. May I know why we need to implicitly define it?

Reply:
ah sorry, you don't need to add this, I just thought that was a reason for failed constrain, no change needed indeed